service/app/org/sunbird/telemetry/actor/TelemetryManagerActor.java

```diff
@@ -0,0 +1,93 @@
+package org.sunbird.telemetry.actor;
+
+import akka.actor.ActorRef;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.lang3.StringUtils;
+import org.sunbird.actor.core.BaseActor;
+import org.sunbird.actor.router.ActorConfig;
+import org.sunbird.actor.service.SunbirdMWService;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+import org.sunbird.common.request.Request;
+import org.sunbird.telemetry.dispatcher.EkstepTelemetryDispatcher;
+import util.Constant;
+
+/**
+ * This actor class will receiving telemetry request and write to EKstep.
+ *
+ * @author Mahesh Kumar Gangula
+ */
+@ActorConfig(
+  tasks = {"dispatchtelemetry"},
+  asyncTasks = {}
+)
+public class TelemetryManagerActor extends BaseActor {
+
+  List<String> dispatchers = new ArrayList<String>();
+  private static final String defaultDispacher = "ekstep";
+
+  public TelemetryManagerActor() {
+    getDispatchers();
+  }
+
+  private void dispatch(String dispatcher, Request baseRequest) throws Exception {
+    Request request = getDispatcherRequest(baseRequest, dispatcher);
+    ActorRef actor = (ActorRef) SunbirdMWService.getRequestRouter();
+    if (null != actor) {
+      actor.tell(request, getSelf());
+    }
+  }
+
+  private Request getDispatcherRequest(Request request, String dispatcher) {
+    Request dispRequest = new Request();
+    dispRequest.setRequest(request.getRequest());
+    dispRequest.setOperation(Constant.DISPATCH_TELEMETRY_OPERATION_NAME + "to" + dispatcher);
+    return dispRequest;
+  }
+
+  @Override
+  public void onReceive(Request request) throws Throwable {
+    String operation = request.getOperation();
+    String ekstepStorageToggle = System.getenv(Constant.EKSTEP_TELEMETRY_STORAGE_TOGGLE);
+    if (Constant.DISPATCH_TELEMETRY_OPERATION_NAME.equals(operation)) {
+      Object body = request.get(JsonKey.BODY);
+      Map<String, String[]> headers = (Map<String, String[]>) request.get(Constant.HEADERS);
+      // if ekstep_telemetry_storage_toggle is on then only send the telemetry value to ekstep.
+      if (StringUtils.isNotBlank(ekstepStorageToggle)
```

Comment:
Default behaviour should be to save in Ekstep. Telemetry service like any other micro service needs to have a properties file where defaults can be defined and easily changed.